scripts/duplicate_pr.py

```diff
@@ -0,0 +1,124 @@
+from enum import Enum
+
+import argparse
+import sh
+
+
+class BranchName(Enum):
+    MASTER = "master"
+    FORMPLAYER = "formplayer"
+
+
+def get_git(path=None):
+    return sh.git.bake(_tty_out=False, _cwd=path)
+
+
+def get_target_branch(orig_target_branch:str):
+    if orig_target_branch == BranchName.FORMPLAYER.value:
+        return BranchName.MASTER.value
+    elif orig_target_branch == BranchName.MASTER.value:
+        return BranchName.FORMPLAYER.value
+    else:
+        print("Only following branches are allowed: {}".format(", ".join([key.value for key in BranchName])))
+        exit(1)
+
+
+def git_create_branch(orig_branch_name:str, new_branch_name: str, git=None):
+    git = git or get_git()
+    try:
+        git.checkout(orig_branch_name)
+    except sh.ErrorReturnCode_1 as e:
+        print(red(e.stderr.decode()))
+        exit(1)
+    try:
+        git.checkout('-b', new_branch_name)
+    except sh.ErrorReturnCode_128 as e:
+        print(red(e.stderr.decode()))
+        exit(1)
+
+
+def git_fetch_pr_branch(branch_name:str, git=None):
+    git = git or get_git()
+    # fetch remote branch without switching branches
+    input = "{0}:{0}".format(branch_name)
+    try:
+        git.fetch("origin", input)
+    except sh.ErrorReturnCode_1 as e:
+        print(red(e.stderr.decode()))
+        exit(1)
+
+
+def get_new_commits(base_branch: str, curr_branch:str, git=None):
+    git = git or get_git()
+    base_commit = merge_base_commit(base_branch, curr_branch)
+    recent_commit = latest_commit(curr_branch)
+
+    commits_range = "{}..{}".format(base_commit, recent_commit)
+    interested_commits = git("rev-list", "--no-merges", commits_range).split()
+    return interested_commits
+
+
+def cherry_pick_new_commits(commits:list[str], branch:str, git=None):
+    git = git or get_git()
+    git.checkout(branch)
+    for commits in reversed(commits):
+        git("cherry-pick", commits)
+
+
+def git_push_pr(branch:str, git=None):
+    git = git or get_git()
+    git.push("origin", branch)
```

Comment:
it would be nice to show the output of this command since it will contain the link to create the PR on Github

Reply:
[00dc3c6](https://github.com/dimagi/commcare-core/pull/1254/commits/00dc3c68e10c144bdab0b480989001243565c440)